### PR TITLE
fixes#24

### PR DESCRIPTION
**Pull request by pradeepbhadani**
Add a variable to replace `.lcl` domain. 
Terraform fmt.

@ExpediaInc/apiary-committers

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -127,7 +127,13 @@ variable "remote_metastores" {
...
-  description = "Option to enable creating dns records for remote metastores"
+  description = "Option to enable creating dns records for remote metastores."
   type        = "string"
   default     = ""
 }
+
+variable "domain_name" {
+  description = "Domain name to use for Rout53 entry and service discovery."
+  type        = "string"
+  default     = "lcl"
```
[the remark this answers is not in the document]
Comment (pradeepbhadani) [reply variables.tf]: updated.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | cpu | The number of CPU units to reserve for the Waggle Dance container. | string | `1024` | no |
 | docker_image | Full path Waggle Dance Docker image. | string | - | yes |
 | docker_version | Waggle Dance Docker image version. | string | - | yes |
+| domain_extension | Domain name to use for Rout53 entry and service discovery. | string | `lcl` | no |
```
Comment (massdosage) [inline README.md]: Route 53

File: README.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
...
+| enable_remote_metastore_dns | Option to enable creating dns records for remote metastores. | string | `` | no |
```
Comment (massdosage) [inline README.md]: dns -> DNS

File: variables.tf [hunk 4]
```diff
@@ -127,7 +127,13 @@ variable "remote_metastores" {
 }
 
 variable "enable_remote_metastore_dns" {
-  description = "Option to enable creating dns records for remote metastores"
+  description = "Option to enable creating dns records for remote metastores."
```
Comment (massdosage) [inline variables.tf]: As above

File: variables.tf [hunk 5]
```diff
@@ -127,7 +127,13 @@ variable "remote_metastores" {
...
   type        = "string"
   default     = ""
 }
+
+variable "domain_extension" {
+  description = "Domain name to use for Rout53 entry and service discovery."
```
Comment (massdosage) [inline variables.tf]: As above

File: README.md (hunk 2 above)
Comment (pradeepbhadani) [reply README.md]: fixed in other PR

File: README.md (hunk 3 above)
Comment (pradeepbhadani) [reply README.md]: fixed in other PR

File: variables.tf (hunk 4 above)
Comment (pradeepbhadani) [reply variables.tf]: fixed in other PR

File: variables.tf (hunk 5 above)
Comment (pradeepbhadani) [reply variables.tf]: fixed in other PR

**Pull request merged**